examples/mongodbatlas_stream_processor/main.tf

```diff
@@ -0,0 +1,43 @@
+resource "mongodbatlas_stream_instance" "example" {
+  project_id    = var.project_id
+  instance_name = "InstanceName"
+  data_process_region = {
+    region         = "VIRGINIA_USA"
+    cloud_provider = "AWS"
+  }
+}
+
+resource "mongodbatlas_stream_connection" "example-sample" {
+  project_id      = var.project_id
+  instance_name   = mongodbatlas_stream_instance.example.instance_name
+  connection_name = "sample_stream_solar"
+  type            = "Sample"
+}
+
+resource "mongodbatlas_stream_processor" "stream-processor-example" {
```

Comment:
It would be nice to include one example per stream connection type: Kafka and Cluster

Reply:
there are Kafka and Cluster stream connection examples in the examples for `mongodbatlas_stream_connection`, that is why I tried to keep it simple here in the `mongodbatlas_stream_processor` example

Reply:
I think it would be nice to have them as examples here to show how the `pipeline` can be used with the different `stream_connection` configs.

Reply:
Done!